### PR TITLE
RadioButtonGroup component

### PR DESCRIPTION
**Pull request by pseuyi**
screen shot 2018-04-02 at 4 13 23 pm

[CMOS-3723](https://jira.we.co/browse/CMOS-3723)

needed for announcements form [CMOS-3704](https://jira.we.co/browse/CMOS-3704)

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[ignore]
+.*/node_modules/findup
```
Comment (kangax) [inline .flowconfig]: shouldn't we ignore all of the node_modules?

File: src/components/RadioButtonGroup/RadioButtonGroup.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    );
+  }
+}
+
+RadioButtonGroup.defaultProps = {
+  value: null,
+};
+
+RadioButtonGroup.propTypes = {
+  fancy: PropTypes.bool,
```
Comment (kangax) [inline src/components/RadioButtonGroup/RadioButtonGroup.jsx]: no need for propTypes if we're adding flow

File: yarn.lock [hunk 3]
```diff
@@ -3841,6 +3841,10 @@ flatten@^1.0.2:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/flatten/-/flatten-1.0.2.tgz#dae46a9d78fbe25292258cc1e780a41d95c03782"
 
+flow-bin@^0.69.0:
+  version "0.69.0"
+  resolved "https://registry.yarnpkg.com/flow-bin/-/flow-bin-0.69.0.tgz#053159a684a6051fcbf0b71a2eb19a9679082da6"
+
```
Comment (kangax) [inline yarn.lock]: how come babel-preset-flow is not in a lock file?

File: src/components/RadioButtonGroup/RadioButtonGroup.jsx (hunk 2 above)
Comment (pseuyi) [reply src/components/RadioButtonGroup/RadioButtonGroup.jsx]: agree, had to add for linter

File: yarn.lock (hunk 3 above)
Comment (pseuyi) [reply yarn.lock]: not sure..

**Pull request merged**